v2/pkg/delete/delete_images.go

```diff
@@ -0,0 +1,331 @@
+package delete
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/openshift/oc-mirror/v2/pkg/api/v1alpha2"
+	"github.com/openshift/oc-mirror/v2/pkg/api/v1alpha3"
+	"github.com/openshift/oc-mirror/v2/pkg/archive"
+	"github.com/openshift/oc-mirror/v2/pkg/batch"
+	"github.com/openshift/oc-mirror/v2/pkg/image"
+	clog "github.com/openshift/oc-mirror/v2/pkg/log"
+	"github.com/openshift/oc-mirror/v2/pkg/manifest"
+	"github.com/openshift/oc-mirror/v2/pkg/mirror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+type DeleteImages struct {
+	Log              clog.PluggableLoggerInterface
+	Opts             mirror.CopyOptions
+	Batch            batch.BatchInterface
+	Blobs            archive.BlobsGatherer
+	Config           v1alpha2.ImageSetConfiguration
+	Manifest         manifest.ManifestInterface
+	LocalStorageDisk string
+	LocalStorageFQDN string
+}
+
+// WriteDeleteMetaData
+func (o DeleteImages) WriteDeleteMetaData(images []v1alpha3.CopyImageSchema) error {
+	o.Log.Info("writing delete metadata images to %s ", o.Opts.Global.WorkingDir+deleteDir)
+
+	// we write the image and related blobs in yaml format to file for further processing
+	filename := filepath.Join(o.Opts.Global.WorkingDir, deleteImagesYaml)
+	discYamlFile := filepath.Join(o.Opts.Global.WorkingDir, discYaml)
```

Comment:
you mean deleteImageSetConfig? :laughing: I was so confused with a disc :cd: 

I don't think the final / is necessary for filepath.Join